admin/build-doc

```diff
@@ -68,19 +68,19 @@ install -d -m0755 \
 # create a dummy librbd.so that allows the module to be imported by sphinx.
 mkdir -p $vdir/lib
 export LD_LIBRARY_PATH="$vdir/lib"
-export PYTHONPATH=`pwd`/../src/pybind
+export PYTHONPATH=$TOPDIR/src/pybind
 ln -sf librbd.so.1 $vdir/lib/librbd.so
 gcc -shared -o $vdir/lib/librbd.so.1 -xc /dev/null
-CFLAGS="-iquote `pwd`/../src/include" \
-    CPPFLAGS="-iquote `pwd`/../src/include" \
+CFLAGS="-iquote $TOPDIR/src/include" \
+    CPPFLAGS="-iquote $TOPDIR/src/include" \
     LDFLAGS="-L$vdir/lib -Wl,--no-as-needed" \
-    $vdir/bin/pip install `pwd`/../src/pybind
+    $vdir/bin/pip install $TOPDIR/src/pybind
 nm $vdir/lib/python*/*-packages/rbd.so | grep 'U rbd_' | \
     awk '{ print "void "$2"(void) {}" }' | \
     gcc -shared -o $vdir/lib/librbd.so.1 -xc -
 
-$vdir/bin/sphinx-build -a -n -b dirhtml -d doctrees ../doc output/html
-$vdir/bin/sphinx-build -a -b man -d doctrees ../doc output/man
+$vdir/bin/sphinx-build -a -n -b dirhtml -d doctrees $TOPDIR/doc $TOPDIR/output/html
+$vdir/bin/sphinx-build -a -b man -d doctrees $TOPDIR/doc $TOPDIR/output/man
```

Comment:
output is a subdir of build-doc - should be $TOPDIR/build-doc/output for both of these